src/main/webapp/app/exam/exam-scores/exam-scores.component.html

```diff
@@ -304,10 +304,28 @@ <h4>
             </tbody>
         </table>
 
-        <div style="display: block">
+        <div #containerRef style="display: block">
             <!-- Histogram -->
-            <canvas baseChart height="400" [datasets]="barChartData" [labels]="barChartLabels" [options]="barChartOptions" [legend]="barChartLegend" [chartType]="barChartType">
-            </canvas>
+            <ngx-charts-bar-vertical
+                [view]="[containerRef.offsetWidth, 400]"
+                [results]="ngxData"
+                [scheme]="ngxColor"
+                [yAxis]="true"
+                [xAxis]="true"
+                [yAxisLabel]="yAxisLabel"
+                [showYAxisLabel]="true"
+                [xAxisLabel]="xAxisLabel"
+                [showXAxisLabel]="true"
+                [yScaleMax]="yScaleMax"
+                [dataLabelFormatting]="dataLabelFormatting"
+                [showDataLabel]="true"
+            >
+                <ng-template #tooltipTemplate let-model="model">
+                    <b>{{ model.name }}</b
```

Comment:
same nitpick here (with the strange breaks)

Reply:
Same case as for the other adjustment above.